cognitiveservices/data-plane/pom.xml

```diff
@@ -38,9 +38,9 @@
     <azure.annotations.version>1.4.0</azure.annotations.version>
     <commons.io.version>2.4</commons.io.version>
     <commons.codec.version>1.10</commons.codec.version>
+    <jackson.version>2.9.7</jackson.version>
```

Comment:
Please update to 2.9.9 (https://dev.azure.com/sqlclientdrivers-ci/mssql-jdbc/_componentGovernance/112445/alert/1551223?typeId=764536)

Reply:
I cannot access that line but I do think it is probably best to bump this to the latest.